### ISSUE TITLE
TYPO in doc

### ISSUE THREAD

**Issue by Doc73** [maieul/ledmac#766]
Issue can be opened in English or in French (I prefer in French). Please fill this form.
Please open one issue by problem, and not a global issue for multiple problems. 

# Version of reledmac / reledpar (if that is a devellopement version, please precise the branch)
2.26.10/2.21.8

# If this is a newly introduced problem, please indicate the last working version of the package(s)
I dont' know

# Does the problem happen only when combining with some packages?
No

# Does the problem happens both in parallel typesetting and normal typesetting?
No

# Description of the problem
Reledmac manual, pag. 59 § 11.1: `\annotref` instead of `\annotationref`.

I also would suggest to add some example, expecially for `\edlabel`: it isn't clear to me where to put the label. You write 

> or  in the first argument of `\edtext`

but `\edtext{gloria\edlabel{fran:gloria}}{\Bfootnote{blablabla}}` isn't working. Where am I wrong?

# Minimal Working Example
I think it isn't compulsory.

**Comment by maieul** [comment]
for the typo, it is fixed. But for your problem with edlabel, I don't understand your problems. 

Cf 
https://github.com/maieul/ledmac/blob/e67b128710b762cdce6fbae10a62dea66f957f03/issue766.tex

which works perfectly. So i suppose you have a more complexe case.

**Comment by Doc73** [comment]
I'm sorry!
It is `\annotationref` that doesn't work!

**Comment by maieul** [comment]
Sorry, but still worjing in my computer.

**Comment by maieul** [comment]
https://github.com/maieul/ledmac/blob/issue766/issue766.tex

**Comment by Doc73** [comment]
Bah! It is not important. If i'll find some free time, I'll send a MWE.
For now, I'll use the combination of `\edlineref` and `\edpageref`, which fits my needs.

**Comment by maieul** [comment]
ok. but that is strange, indeed. maybe are you in nested edtext. 

In any case, please open a new issue with your mwe. 

I wont publish the correction for the typo now. Maybe in june if I don't do any new release before.

**Comment by Doc73** [comment]
Ah! It is in nested extext!
